Mathlib/Tactic/Says.lean

```diff
@@ -30,12 +30,15 @@ open Lean.Meta.Tactic.TryThis
 
 namespace Mathlib.Tactic.Says
 
+/-- For every appearance of the `X says Y` combinator,
+re-verify that running `X` produces `Try this: Y`. -/
 register_option says.verify : Bool :=
   { defValue := false
     group := "says"
     descr := "For every appearance of the `X says Y` combinator, \
```

Comment:
Do we have an elaborator that avoids needing to repeat the docstring here? Can you ask on Zulip?

Reply:
How about I change their docstrings to
```lean
/-- If this option is `true`, verify for `X says Y` that `X says` outputs `Y`. -/
register_option says.verify : Bool :=
  { defValue := false
    group := "says"
    descr := "Verify the output" }

/-- This option is only used in CI to negate `says.verify`. -/
register_option says.no_verify_in_CI : Bool :=
  { defValue := false
    group := "says"
    descr := "Disable reverification, even if the `CI` environment variable is set." }
```